apps/web/src/server/bases/getBasesList.ts

```diff
@@ -40,10 +40,7 @@ const getWhereBasesProfileList = (
   user?: Pick<SessionUser, 'id'> | null,
 ) =>
   getWhereBasesList(user, {
-    OR: [
-      { createdById: profileId },
```

Comment:
We want to rely only only base where user is a member, even if it's the creator. 
Leaving a base means only to not be part of it, but you remains the creator, so we can keep track of a creation history doing that